apps/aot/src/app/globals.css

```diff
@@ -22,7 +22,7 @@
     --border: 240 5.9% 90%;
     --input: 240 5.9% 90%;
 
-    --primary: 211 61% 48%;
+    --primary: 211 61% 47%;
```

Comment:
The color contrast for this blue is _really_ close to the minimum 4.5:1. But it's not quite there:

* Before (4.35:1): https://webaim.org/resources/contrastchecker/?fcolor=FAFAFA&bcolor=3078C5
* After (4.53:1): https://webaim.org/resources/contrastchecker/?fcolor=FAFAFA&bcolor=2F75C1

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<img width="99" alt="Screenshot of before 'Submit' button with just barely not dark enough blue for contrast" src="https://github.com/user-attachments/assets/3c94b760-bc91-44da-a351-cd9701b3b334">
</td>
<td>
<img width="99" alt="Screenshot of after 'Submit' button with just barely dark enough blue for contrast" src="https://github.com/user-attachments/assets/5de49c83-de4c-4a80-bf49-d66979fe9a31">
</td>
</tr>
</tbody>
</table>